addons/web/static/src/core/browser/router.js

```diff
@@ -132,6 +162,14 @@ export function startRouter() {
     _lockedKeys = new Set(["debug"]);
 }
 
+registry.category("getRoute").addEventListener("UPDATE", () => {
```

Comment:
maybe dont startRouter at the begining of the module, but at the start of the webclient